akka-actor/src/main/scala/akka/util/Unused.scala

```diff
@@ -12,6 +14,15 @@ import akka.annotation.InternalApi
  * or other reason. Useful in combination with
  * `-Ywarn-unused:explicits,implicits` compiler options.
  *
+ * Extends 'deprecated' to make sure using a parameter marked @unused
+ * produces a warning, and not using a parameter marked @unused does not
+ * produce an 'unused parameter' warning.
+ *
+ * Extending 'deprecated' is deprecated in Scala 2.13 and scheduled to be
+ * removed in 2.14. Perhaps we should promote introducing an `@unused`
+ * to Scala? https://contributors.scala-lang.org/t/more-error-reporting-annotations/1681/7
```

Comment:
These to new paragraphs read as redundant slightly, can we compress and remove one of the `Extending 'deprecated'`?